test/docker-compose-template.yml

```diff
@@ -0,0 +1,4 @@
+version: "3.9"
```

Comment:
I'd rename this to `docker-compose.yml` and add a comment on top describing what it's for and how it can be adjusted.

Reply:
Makes sense. :+1: